src/scene/Scale.js

```diff
@@ -162,17 +162,23 @@ function ordinal(scale, rng, group) {
     scale.rangeBands(rng, pad, outer);
   }
 
-  if (!scale.invert) {
-    scale.invert = function(x, y) {
+  prev.range = rng;
```

Comment:
Not sure I follow this line. If `prev` is the previous scale reference, this is overwriting its `range` method. But maybe `prev` is something else?